packages/kit/src/core/adapter-utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Decides how the body should be parsed based on its mime type.
+ *
+ * This is intended to be used with both requests and responses, to have a consistent body parsing across adapters.
+ *
+ * @param {string} content_type The `content-type` header of a request/response.
+ * @returns {boolean}
+ */
+export function isContentTypeBinary(content_type) {
```

Comment:
The logic is possibly still too blunt. There are at least a few content types I'd consider common for which I think this would return the wrong result:
* `image/svg+xml`
* `application/zip`
* `application/pdf`

An escape hatch for the developer here might be useful too given all the potential edge cases that the wonderful world of mime types involves.

Reply:
I know that's why I made #1829

Reply:
I sent https://github.com/sveltejs/kit/pull/1890 to help with this